[New] Apply renderers to scene layer (#703)
Esri/arcgis-maps-sdk-swift-samples: Shared/Samples/Apply renderers to scene layer/README.md

# Apply renderers to scene layer

Change the appearance of a 3D object scene layer with different renderers.

![Screenshot of apply renderers to scene layer sample](apply-renderers-to-scene-layer.png)

## Use case

> A scene layer of 3D buildings hosted on ArcGIS Online comes with a preset renderer that defines how the buildings are
> displayed in the application. However, the fill color may sometimes blend into the basemap, making the buildings
> difficult to distinguish. To enhance visualization, you can apply a custom renderer with a more contrasting fill color,
> helping the 3D buildings stand out more clearly. Additionally, you can use a unique value renderer to represent
> different building uses, or a class breaks renderer to visualize building ages - valuable insights for urban planning
> and analysis.

Reviewer: ```suggestion
A scene layer of 3D buildings hosted on ArcGIS Online comes with a preset renderer that defines how the buildings are displayed in the application. However, the fill color may sometimes blend into the basemap, making the buildings difficult to distinguish. To enhance visualization, you can apply a custom renderer with a more contrasting fill color, helping the 3D buildings stand out more clearly. Additionally, you can use a unique value renderer to represent different building uses, or a class breaks renderer to visualize building ages - valuable insights for urban planning and analysis.
```
I didn't notice the design has linebreak… 🤥

## How to use the sample

> Wait for the scene layer to load. The original scene layer displays 3D textured buildings. Tap on the "Select Renderer"
> dropdown menu and choose a different renderer to change how the buildings are visualized. Each renderer applies
> different symbology to the scene layer. Setting the renderer to null will remove any applied symbology, reverting the
> buildings to their original textured appearance.

Reviewer: ```suggestion
Wait for the scene layer to load. The original scene layer displays 3D textured buildings. Tap on the renderer picker and choose a different renderer to change how the buildings are visualized. Each renderer applies different symbology to the scene layer. Setting the renderer to `nil` will remove any applied symbology, reverting the buildings to their original textured appearance.
```

## How it works

1. Create an `ArcGISSceneLayer` from a service URL.

> 2. Add the scene layer to an `ArcGISScene` and display it in a `SceneView`.

Reviewer: ```suggestion
2. Add the scene layer to an `Scene` and display it in a `SceneView`.
```
